### DISCUSSION TITLE
cidr scan

### DISCUSSION THREAD

**Discussion by molitona** [projectdiscovery/naabu#427] [Q&A] [answered]
Hi, can naabu scan a cidr x.x.x.x/xx ?
if so how the command looks like ?

**Comment by ehsandeep** [comment, marked as answer]
```
naabu -host 173.0.84.0/24 -p 80
```

@molitona, you can feed CIDR as `host` ot `list` option.

**Comment by molitona** [comment]
thank you so much!!
